src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        Bulldozer bulldozer3 = new Bulldozer();
```

Comment:
If the `Bulldozer` class has a constructor that requires parameters or if there are attributes that need to be initialized, you should provide those values here. Otherwise, these instances will be created with default values, which might not be intended.